lua/easy-dotnet/options.lua

```diff
@@ -137,14 +136,27 @@ local M = {
     },
     csproj_mappings = true,
     fsproj_mappings = true,
-    auto_bootstrap_namespace = true,
-  }
+    auto_bootstrap_namespace = {
+      type = "block_scoped",
+      enabled = true
+    },
+  },
 }
 
 local function merge_tables(default_options, user_options)
   return vim.tbl_deep_extend("keep", user_options, default_options)
 end
 
+--- Auto_bootstrap namespace can be either true or table with config
+local function handle_auto_bootstrap_namespace(a)
+  if type(a.auto_bootstrap_namespace) ~= "table" then
+    a.auto_bootstrap_namespace = {
+      type = "block_scoped",
+      enabled = a.auto_bootstrap_namespace == true
+    }
+  end
+end
+
 M.set_options = function(a)
   M.options = merge_tables(M.options, a or {})

```

Comment:
missing the handle_auto_bootstrap_namespace here

Reply:
Oh crap. Yeah I’ll fix that and those lint updates updates when I get back home.

Reply:
No rush! Awesome work!♥

Reply:
good morning! should be ok to merge now